portfolio/src/main/java/com/google/sps/servlets/ShawarmaServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.ShawarmaPlace;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Servlet that returns list of shawarma places.
+ */
+@WebServlet("/shawarma")
+public class ShawarmaServlet extends HttpServlet {
+    private final String SHAWARMA_FILE = "shawarma.json";
+    private final Gson gson = new Gson();
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("application/json");
+        try (final Reader reader = Files.newBufferedReader(Paths.get(
+                        Thread.currentThread().getContextClassLoader().getResource(SHAWARMA_FILE).toURI()))) {
```

Comment:
do you really need to use `Thread.currentThread().getContextClassLoader().getResource(SHAWARMA_FILE).toURI()` here? What does `Paths.get(SHAWARMA_FILE)` return? Can we build a relative path from that instead? Also, this path is unlikely to change, so maybe we can extract it to the constant. Instead of having `String SHAWARMA_FILE = "shawarma.json"`, you could have something like `Path SHAWARMA_JSON_PATH = Paths.get(..., "shawarma.json")`.